plugins/woocommerce/client/blocks/assets/js/blocks/checkout/inner-blocks/checkout-order-summary-block/frontend.tsx

```diff
@@ -100,7 +96,7 @@ const FrontendBlock = ( {
 				</div>
 			</div>
 
-			{ ! isLarge && (
+			{ ( isMedium || isSmall || isMobile ) && (
```

Comment:
Reverted to JS handling of the CheckoutOrderSummaryFill
Using the reverted condition, to make sure this doesn't get rendered before we have a value for isLarge

Reply:
<img width="905" alt="Screenshot 2025-06-06 at 19 18 45" src="https://github.com/user-attachments/assets/1e0a0e78-607c-49d9-90a2-a95cbf876b7f" />
As discussed, I considered rendering the <OrderMetaSlotFill /> from line 99 (see image) only on desktop, so we don't have the same Slot twice in the page. 

But I decided against it because:
- it means the component still unmounts and remounts when switching between large and small screens
- it can lead to potential memory leaks if cleanup isn't handled properly
- it might impact the functionality of some plugins, which work in the current setup